### DISCUSSION TITLE
Finding a Good first Issue

### DISCUSSION THREAD

**Discussion by TakangNina** [aircrack-ng/aircrack-ng#2624] [Q&A] [answered]
Hello everybody, I am Nina Takang and I am new to open source and I would like to be a contributor on this project. Problem is I couldn't find any labels flagged as "good first issues". I need help making my first contribution please. Looking forward to get a reply, thank you

**Comment by Mister-X-** [comment]
What are your skills, and what interests you?

That will help me find one or more issues. And there are things outside of the issues (such as documentation, etc) as well.

**Reply by TakangNina** [reply, marked as answer]
I have been a javascript for the most part of my life, but then 10months ago I diverted to learning ethical hacking hence I am familiar with these tools, I have been learning C for a while and thought it would be nice to contribute to aircrack-ng since I am quite familiar with it. But then I totally buy your idea with perhaps maybe starting small and walking my way from the documentations? I spent some time reading the documentations and I found some words which could use some changes
